README.md

```diff
@@ -1 +1,27 @@
-# CFReT_data
\ No newline at end of file
+# CFReT Project
+
+## Data
+
+The data used in this project is a [Cell Painting assay](https://www.moleculardevices.com/applications/cell-imaging/cell-painting#gref) on [cardiac fibroblasts](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5588900/#:~:text=Definition%20by%20function,%2C%20and%20glycoproteins5%2C6.) from 3 patients that suffered from cardiac arrest. 
+
+In this Cell Painting, there are five channels:
+
+- `d0` (Nuclei)
```

Comment:
Have you seen documentation to indicate these structures? This doesn't seem to me to be the standard Cell Painting Assay 

See Table 1 of [Bray et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5223290/)

Reply:
@gwaybio What I have is how the documentation we have gotten for this project states how the channels are structured and what is being stained for in each. I can see that this does not follow the table given in [Bray et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5223290/), but I am following the figure below.

![CFReT_channels](https://user-images.githubusercontent.com/107513215/190666609-ad0de89d-145d-4770-8421-2887f46bbf76.png)

Reply:
We'll probably want to add this figure somewhere, at some point! (probably the main readme!)